recipes/quantecon/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = "0.3.3" %}
+{% set sha256 = "0aa5f4d3d4b6a92982fb4172f5c1d8c4e1af7abf185b23f103a3ed95520c5730" %}
+
+package:
+  name: quantecon
+  version: {{ version }}
+
+source:
+  fn: QuantEcon.py-{{ version }}.tar.gz
+  url: https://github.com/QuantEcon/QuantEcon.py/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+    - numpy
+    - scipy
+    - matplotlib
+    - pandas
+    - sympy
+    - numba
+    - requests
+    - nose
+    - pytables
+
+
+test:
+  imports:
+    - quantecon
+    - quantecon.tests
+    - quantecon.game_theory
+    - quantecon.markov
+    - quantecon.random
+    - quantecon.util
+
+about:
+  home: https://github.com/QuantEcon/QuantEcon.py
+  license: BSD-3-Clause
+  summary: 'QuantEcon is a package to support all forms of quantitative economic modelling.'
+  description: |
+    **QuantEcon** is an organization run by economists for economists with the aim of coordinating
+    distributed development of high quality open source code for all forms of quantitative economic modelling.
+    The project website is located at `http://quantecon.org/ <http://quantecon.org/>`_. This website provides
+    more information with regards to the **quantecon** library, documentation, in addition to some resources
+    in regards to how you can use and/or contribute to the package.
+    The **quantecon** Package
+    -------------------------
+    The `repository <https://github.com/QuantEcon/QuantEcon.py>`_ includes the Python package ``quantecon``
+    Assuming you have `pip <https://pypi.python.org/pypi/pip>`_ on your computer --- as will be the case if you've `installed Anaconda <http://quant-econ.net/getting_started.html#installing-anaconda>`_ --- you can install the latest stable release of ``quantecon`` by typing
+        pip install quantecon
+    at a terminal prompt
+    **Note:** There is also a Julia version available for Julia users `QuantEcon.jl <https://github.com/QuantEcon/QuantEcon.jl>`_
+  doc_url: http://quanteconpy.readthedocs.io/en/latest/
+  dev_url: https://github.com/QuantEcon/QuantEcon.py
+
+extra:
+  recipe-maintainers:
+    - mmcky
+
+# See
+# http://docs.continuum.io/conda/build.html for
+# more information about meta.yaml
```

Comment:
@mmcky just remove the remaining boilerplate comments from `conda-skeleton` and this is good to go!